libs/shared/ofrep-core/src/lib/model/evaluation.ts

```diff
@@ -48,23 +52,15 @@ export function isEvaluationSuccessResponse(response: unknown): response is Eval
   return 'value' in response;
 }
 
-export enum EvaluationFailureErrorCode {
```

Comment:
This is a duplicate of some SDK constants, we don't need it and it save us tedious mapping.